runtime/binding-http-kafka/src/test/java/io/aklivity/zilla/runtime/binding/http/kafka/internal/config/HttpKafkaWithConfigAdapterTest.java

```diff
@@ -129,11 +131,13 @@ public void shouldWriteWithFetchTopicAndFilters()
         HttpKafkaWithConfig with = HttpKafkaWithConfig.builder()
             .fetch(HttpKafkaWithFetchConfig.builder()
                 .topic("test")
-                .filters(singletonList(new HttpKafkaWithFetchFilterConfig(
-                        "fixed-key",
-                        singletonList(new HttpKafkaWithFetchFilterHeaderConfig(
-                            "tag",
-                            "fixed-tag")))))
+                .filters(singletonList(HttpKafkaWithFetchFilterConfig.builder()
+                    .key("fixed-key")
+                    .headers(singletonList(HttpKafkaWithFetchFilterHeaderConfig.builder()
```

Comment:
There should also be a `.header(String name, String value)` method on the builder that lazily instantiates the `headers` list and adds the header config.